pkg/server/grpc/grpc.go

```diff
@@ -79,6 +81,8 @@ func New(logger log.Logger, reg prometheus.Registerer, tracer opentracing.Tracer
 	storepb.RegisterStoreServer(s, storeSrv)
 	met.InitializeMetrics(s)
 
+	grpc_health.RegisterHealthServer(s, health.NewServer())
```

Comment:
Doesn't it mean that we would always be reporting the health as `SERVING`? Maybe it would be a good idea to tie this with the `prober` package?

Reply:
I wanted to keep things simple, I assumed if it’s running it’s serving and it’d be enough. However I get your point I’ll integrate it with prober.